docs/contribute/01_environment.qmd

```diff
@@ -11,14 +11,21 @@ hide:
 
 ::: {.panel-tabset}
 
+::: {.callout-note}
+## MacOS users
```

Comment:
This should be a summary statement like "MacOS users on arm CPUs must use a different setup"